styleguide/components/21-atoms/link/_link.scss

```diff
@@ -6,37 +6,62 @@ a {
   text-decoration: underline;
   cursor: pointer;
```

Comment:
Should happen by default on correct links.
If it doesn't, it's a good indication that your link is not correct (missing a href for instance).

Reply:
Removed it.